pyscriptjs/src/runtime.ts

```diff
@@ -6,7 +6,7 @@ const logger = getLogger('pyscript/runtime');
 
 // VERSION
 // Version number of release
-export const version = '2022.12.1.RC1';
+export const version = '2022.12.1.release';
```

Comment:
are we sure we want the `.release`?
It looks very weird to me, I'd expect just `2022.12.1` (i.e., the same string that we use in the URLs)

Reply:
Here are @JeffersGlass notes...
"Sets the __version__ and version_info properties from provided JSON data
    Args:
        version_from_runtime (str): A "dotted" representation of the version:
            YYYY.MM.m(m).releaselevel
            Year, Month, and Minor should be integers; releaselevel can be any string
    """

Reply:
I propose making the releaselevel optional in the regex tests.

Reply:
Implemented and tests running